functions.h

```diff
@@ -102,6 +102,9 @@ const struct binding_t OpMain[] = { /* map: index */
   { "delete-subthread",		OP_DELETE_SUBTHREAD,		"\033d" },
   { "edit",			OP_EDIT_MESSAGE,		"e" },
   { "edit-label",		OP_EDIT_LABEL,			"Y" },
+#ifdef USE_IMAP
+  { "edit-keywords",		OP_EDIT_KEYWORDS,			NULL },
```

Comment:
could you use a more explicit name, like `imap-edit-keywords`?

On a side note, this makes me realise that `modify-labels` (for notmuch) is not very homogeneous nor namespaced.